lunar/distribution.yaml

```diff
@@ -303,6 +303,25 @@ repositories:
       url: https://github.com/pyros-dev/catkin_pip.git
       version: devel
     status: developed
+  catkin_virtualenv:
+    doc:
+      type: git
+      url: https://github.com/locusrobotics/catkin_virtualenv.git
+      version: devel
+    release:
+      packages:
+      - catkin_virtualenv
+      - test_catkin_virtualenv
+      - test_catkin_virtualenv_py3
```

Comment:
Do you really want these test packages to be available as Debian packages or should they be ignored in the release step?

Reply:
Happy to take them out, will the PR/devel builders still run the tests in those packages even if they're not listed here?

Reply:
> will the PR/devel builders still run the tests in those packages even if they're not listed here?

Yes, the PR/devel jobs are processing the while content of the repo specified under `source`.